### PR TITLE
WIP | Group terraform files into folder

### PR DESCRIPTION
**Pull request by anthonyhashemi**
**Changes in this PR:**

Group terraform files into terraform folder to help navigating the repo more easily.
Have adjusted the reference to main.tf in the readme to point to this folder now.

**Before merging:**
I imagine we need to point something towards the moved main.tf file? So we should check this first.

**_**UPDATE**: Okay so all the builds failed I imagine since the main.tf is no longer being picked up. First time working with TF so do shout if you know what to do - looking at docs in the meantime_**

**Note on terraform lint:**
To make the commit where I moved the .tf files, I had to skip verification with --no-verify since terraform lint was complaining about the following:

Screenshot 2023-01-26 at 16 37 38

Are these things we should be worried about? If not, can someone explain why and help me to add them to the ignore list 🙏 

**Trello Card:**
https://trello.com/c/RqyUEeFz/380-reorganise-enrichment-repo-to-help-navigating-it

### CODE REVIEW

File: terraform/main.tf [hunk 1]
```diff
@@ -18,6 +18,8 @@ terraform {
     region = "eu-west-2"
   }
```
Comment (anthonyhashemi) [inline terraform/main.tf:20]: woops let me remove these comments about comments 😄

File: terraform/main.tf [hunk 2]
```diff
@@ -18,6 +18,8 @@ terraform {
...
 
+# why commented?
```
Comment (anthonyhashemi) [inline terraform/main.tf:21]: woops let me remove these comments about comments 😄

File: terraform/s3.tf [hunk 3]
```diff
@@ -43,3 +43,5 @@
 #   batch_size       = 1
 # }
 
+
+# why is this all commented? can it be removed?
```
Comment (anthonyhashemi) [inline terraform/s3.tf:47]: same here

**Comment by ak-mdrxtech** [comment]
This would fail unless the CICD is amended to look at the new location. The CICD and invoking terraform go hand in hand.

**Comment by anthonyhashemi** [comment]
> This would fail unless the CICD is amended to look at the new location. The CICD and invoking terraform go hand in hand.

yeah I realised I need to either cd into terraform, or adjust terraform commands to be `terraform -chdir terraform .....`

**Comment by anthonyhashemi** [comment]
Closing in favour of new PR due to this one becoming outdated: move all terraform files into a terraform folder: https://github.com/nationalarchives/ds-caselaw-data-enrichment-service/pull/189

**Pull request closed without merging**